### ISSUE TITLE
[BUG] 3 - Landcover query issues [OSDP-1801] (3 layers)

### ISSUE THREAD

**Issue by MLAuger** [Canadian-Geospatial-Platform/geoview#2603]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

The 3 land cover layers can't be queried in RAMP. They can in GeoView but return a strange value: "band-0-pixel-value"

![image](https://github.com/user-attachments/assets/df524183-9006-4739-96aa-81ac40dd7a92)

Concerned UUIDs:
ee1580ab-a23d-4f86-a09b-79763677eb47
c688b87f-e85f-4842-b0e1-a8f79ebf1133
4e615eae-b90c-420b-adee-2ca35896caf6
 

### Steps To Reproduce

Load any of the three UUIDs and click on the data to query once loaded.

**Comment by jolevesq** [comment]
This is returning the WMS value, field name and value. If we do not want it we need to set not query able. I do not think this is a layer issue. Can we close?

```
<FeatureCollection xmlns="http://www.opengis.net/wfs" xmlns:gml="http://www.opengis.net/gml" xmlns:ows="http://www.opengis.net/ows/1.1" xmlns:ogc="http://www.opengis.net/ogc" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><gml:featureMember
><Geometry>
…
</Geometry
><band-0-pixel-value>15.0</band-0-pixel-value></gml:featureMember></FeatureCollection>
```

**Comment by MLAuger** [comment]
You are right, this is not an issue. I had a talk with the Data Cube team this week about this and it does behaves as expected. It's RAMP that did not.
